### ISSUE TITLE
JEI World Loadup issue

### ISSUE THREAD

**Issue by Iskalldren** [mezz/JustEnoughItems#2571]
Been developing my own modpack for 1.16.5 Forge 36.2.4, when I try to load up a new world it gets stuck at the following line according to the logs. 
`[21:50:11] [Render thread/INFO]: JEI Plugin is Registering subtypes`

Here is a picture showing how the logs have been stuck like this for 10 minutes:
https://drive.google.com/file/d/1GNiM9hO49pTnLJx7huLOC0WU51a0_2zC/view?usp=sharing

latest.log:
https://drive.google.com/file/d/14Y4PRCco_NVIgsc7CETfrZVzrbKWb34e/view?usp=sharing

**Comment by mezz** [comment]
Thanks for the report!
The log line
> [21:50:11] [Render thread/INFO]: JEI Plugin is Registering subtypes

is actually not from JEI, it is from some mod's JEI Plugin.
Unfortunately I am not sure which mod is causing the issue, you may need to remove mods to find which one is causing a problem.

**Comment by Iskalldren** [comment]
If it's any help, this issue seems to be fairly inconsistent. I myself am on an M1 MacBook pro and I sent the mod pack I was using to my Windows PC friends and they seem to work fine. Also, sometimes it loads and sometimes it doesn't and it requires me to start and relaunch my Minecraft launcher. Could it be an issue with the compatibility of the new M1 macs with Minecraft, specifically modded? It also seems to be purely client-side, because we set up a server and I managed to physically join the server just on the client side I am stuck hanging on the dirt screen.
